skimage/transform/_warps.py

```diff
@@ -134,11 +139,27 @@ def resize(image, output_shape, order=1, mode=None, cval=0, clip=True,
                    mode=mode, cval=cval, clip=clip,
                    preserve_range=preserve_range)
 
+    else:  # n-dimensional interpolation
+        coord_arrays = [dim_scales[i] * (np.arange(d) + 0.5) - 0.5
+                        for i, d in enumerate(output_shape)]
+
+        coord_map = np.array(np.meshgrid(*coord_arrays,
+                                         sparse=False,
+                                         indexing='ij'))
+
+        image = convert_to_float(image, preserve_range)
+
+        ndi_mode = _to_ndimage_mode(mode)
+        out = ndi.map_coordinates(image, coord_map, order=order,
+                                  mode=ndi_mode, cval=cval)
+
+        _clip_warp_output(image, out, order, mode, cval, clip)
+
     return out
 
 
 def rescale(image, scale, order=1, mode=None, cval=0, clip=True,
```

Comment:
Please, update this docstring as well:
`(row_scale, col_scale)` -> nD notation.